bears/python/YapfBear.py

```diff
@@ -0,0 +1,20 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='yapf',
+        use_stdin=True,
+        output_format='corrected',
+        result_message='Formatting can be improved.')
+class YapfBear:
+    """
+    Check and correct formatting of python Code using yapf utility.
+    """
+    LANGUAGES = {"Python"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
```

Comment:
I think it'd be nice to make it configurable right ahead: https://github.com/google/yapf#usage